weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/SimpleFilterableDataTable.tsx

```diff
@@ -185,28 +192,47 @@ export const FilterableTable = <
     );
   }, [effectiveFilter, filter, filteredColData, props.columns, setFilter]);
 
+  // Highlight table row if it matches peek drawer.
+  const query = useURLSearchParamsDict();
```

Comment:
something like `useGetPeekId` could reduce the duplication in the 3 sections.

Reply:
Good idea, will note for follow up